apm-sniffer/apm-sdk-plugin/hystrix-1.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/hystrix/v1/define/HystrixPluginsInstrumentation.java

```diff
@@ -54,6 +56,19 @@
                 @Override public boolean isOverrideArgs() {
                     return false;
                 }
+            },
+            new InstanceMethodsInterceptPoint() {
+                @Override public ElementMatcher<MethodDescription> getMethodsMatcher() {
+                    return named(GET_CONCURRENCY_STRATEGY_METHOD);
+                }
+
+                @Override public String getMethodsInterceptor() {
+                    return GET_CONCURRENCY_STRATEGY_INTERCEPT_CLASS;
+                }
+
+                @Override public boolean isOverrideArgs() {
+                    return true;
```

Comment:
Override=`true`? Are you sure? why? I just saw you wrap the result, not override argument.